app/src/main/java/com/hamdan/acceleromatorminiproject/data/AccelerationDao.kt

```diff
@@ -0,0 +1,33 @@
+package com.hamdan.acceleromatorminiproject.data
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface AccelerationDao {
+
+    @Insert
+    suspend fun insert(scoreEntity: ScoreEntity)
+
+    @Query(
+        """
+        DELETE FROM ${ScoreEntity.SCORE_TABLE_NAME}
+        """
+    )
+    suspend fun nukeTable()
```

Comment:
is this needed?

Reply:
Not in use, but included for future usage

Removed from this PR